prober/dns.go

```diff
@@ -141,6 +141,16 @@ func ProbeDNS(ctx context.Context, target string, module config.Module, registry
 	registry.MustRegister(probeDNSAuthorityRRSGauge)
 	registry.MustRegister(probeDNSAdditionalRRSGauge)
 
+	qc := uint16(dns.ClassINET)
+	if module.DNS.QueryClass != "" {
+		var ok bool
+		qc, ok = dns.StringToClass[module.DNS.QueryClass]
+		if !ok {
+			level.Error(logger).Log("msg", "Invalid query class", "Class seen", module.DNS.QueryClass, "Existing classes", dns.ClassToString)
```

Comment:
This should be done in the config so that it can be caught when the config is loaded.

Reply:
Would that mean the [QueryType check L148](https://github.com/prometheus/blackbox_exporter/pull/635/files/1f04ae59feacb65cd358ff52a665b9e2da193dc6#diff-26673a9bcc326ad193c4d9832bfc4798L148) (should be also done in the config?
Would you accept a 2nd PR to move both checks outside this file?

Reply:
Indeed, it should be. It's a small enough change it can all be done here.

Reply:
done, sorry for the delay.
I added config checks but also kept the existing code in the prober.